contracts/auction-handler/FastLaneAuctionHandler.sol

```diff
@@ -96,6 +96,9 @@ contract FastLaneAuctionHandler is FastLaneAuctionHandlerEvents {
     /// @notice The scale for validator refund share
     uint256 internal constant VALIDATOR_REFUND_SCALE = 10_000; // 1 = 0.01%
 
+    /// @notice The default refund share for validators
+    int256 internal constant DEFAULT_VALIDATOR_REFUND_SHARE = 5000; // 50%
```

Comment:
```suggestion
    uint256 internal constant DEFAULT_VALIDATOR_REFUND_SHARE = 5000; // 50%
```
Is there a reason why int is used instead of uint? Doesn't make much difference here, but in the mapping it feels like its implying a negative validator refund share is possible